classes/item.rb

```diff
@@ -0,0 +1,32 @@
+require 'date'
+
+# Item class and defines the attributes of an item
+class Item
+  attr_accessor :publish_date, :label, :genre, :archived
+  attr_reader :author
+
+  def initialize(publish_date)
+    Random.rand(1..1_000_000)
+    @publish_date = publish_date
+    @archived = false
+  end
+
+  def move_to_archive?
+    @archived = true if can_be_archived?
+    @archived
+  end
+
+  def author=(author)
+    @author = author
+    author.items.push(self) unless author.items.include?(self)
+  end
+
```

Comment:
- [ ] The item class should include the `Genre`, `Author`, and `Label` methods respectively. Please add them to your item class just as you added the author method.

Reply:
Hello @emmanuellekamwa 
Appreciate the feedback, we are working on it.
Thank you

Reply:
Resolved